src/main/java/org/lantern/Launcher.java

```diff
@@ -279,26 +279,11 @@ void launch() {
             }
         }
         
-        proxyTracker = instance(ProxyTracker.class);
+        //proxyTracker = instance(ProxyTracker.class);
         httpClientFactory = instance(HttpClientFactory.class);
 
         s3ConfigFetcher = instance(S3ConfigFetcher.class);
         
-        if (checkFallbacks) {
```

Comment:
The check-fallbacks mode is used to test the fallback servers, which we still use.